src/Health/Router.ts

```diff
@@ -0,0 +1,48 @@
+import { inject, injectable } from 'inversify'
+import * as Pino from 'pino'
+
+import { childWithFileName } from 'Helpers/Logging'
+import { Messaging } from 'Messaging/Messaging'
+
+import { ExchangeConfiguration } from './ExchangeConfiguration'
+import { HealthController } from './HealthController'
+
+@injectable()
+export class Router {
+  private readonly logger: Pino.Logger
+  private readonly messaging: Messaging
+  private readonly controller: HealthController
+  private readonly exchange: ExchangeConfiguration
+
+  constructor(
+    @inject('Logger') logger: Pino.Logger,
+    @inject('Messaging') messaging: Messaging,
+    @inject('HealthController') controller: HealthController,
+    @inject('ExchangeConfiguration') exchange: ExchangeConfiguration
+  ) {
+    this.logger = childWithFileName(logger, __filename)
+    this.messaging = messaging
+    this.controller = controller
+    this.exchange = exchange
+  }
+
+  async start() {
+    await this.messaging.consume(this.exchange.getHealth, this.onGetHealth)
+  }
+
+  async stop() {
```

Comment:
It's preferred for the root of the module to do the `this.messaging.stop()`, since it's the one doing the `messaging.start()`.

The reason our RMQ routers have a `start` but no `stop` is because [we're not cancelling channels](https://www.squaremobius.net/amqp.node/channel_api.html#channel_cancel).

Reply:
Ok. Should I keep?:
```js
async stop() {
    this.logger.info('Stopping Health Messaging...')
  }
```